grobid-core/src/main/java/org/grobid/core/main/LibraryLoader.java

```diff
@@ -27,89 +27,92 @@
     private static boolean isContextMocked = false;
 
     public static void load() {
-        if (!loaded) {
```

Comment:
I inverted the if statement to reduce nesting, but the diff on GitHub is a mess.